tests/integration/models_containers_test.py

```diff
@@ -360,7 +360,7 @@ def test_exec_run_failed(self):
         )
         self.tmp_containers.append(container.id)
         exec_output = container.exec_run("docker ps")
-        assert exec_output[0] == 126
+        assert exec_output[0] == 127
```

Comment:
Perhaps check for "either 126 or 127" and a comment?

Given that this test was added as a general "include exit code", also wondering if something like this would work (produce a custom exit code, and check for that)?

```bash
docker exec foo sh -c 'exit 42'
echo $?
42
```

Reply:
I _this_ test is just for the exit code for a failed exec (as in, failed to start), not for general exit code. it fails that way because `docker` isn't a binary in the container. The exit code here isn't the exit code from the process executed in the container as a general "include exit code" test because that can only happen if the process in the container can be started.

I'll TAL and see if we're missing tests for that case though, and can add some.